project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
-addSbtPlugin("org.allenai.plugins" % "allenai-sbt-plugins" % "1.5.0")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.4")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.1")
+addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
```

Comment:
How come these changed?

Reply:
PGP used for the sbt-release process. The Bintray release process doesn't seem to sign and instead relies on our local Bintray credentials. I'm not sure beyond that (admittedly this is a bit of a disappointing stopping point). However, it works, and I didn't want to spend too much time on this as the recommendation seems to be to use bintray for releases.